src/index.html

```diff
@@ -1,14 +1,48 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:500&subset=cyrillic-ext" rel="stylesheet">
+  </head>
+  <body>
+    <header class="head">
+      <a href="#" class="logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class = "navlist">
+        <ul class="nav">
+          <li class = "nav_point is-active point_1">
```

Comment:
```suggestion
          <li class="nav_point is-active point_1">
```
Тут не нужны пробелы